README.md

```diff
@@ -42,12 +42,19 @@ To build the plugin and run the tests, you will need Java 11.
 ./gradlew clean build
 ```
 
-If you face an error related to dependencies checksums, you can update `gradle/verification-metadata.xml` and review it.
+#### Fix the error `* What went wrong: Dependency verification failed ... update the gradle/verification-metadata.xml file ...`
+You need to update `gradle/verification-metadata.xml` and review it.
 ```bash
-./gradlew build --write-verification-metadata sha256
+./gradlew --refresh-dependencies --info --stacktrace --write-verification-metadata sha256
 git diff -- gradle/verification-metadata.xml
 ```
 Once you have reviewed the changes, replace `origin="Generated by Gradle"` with `origin="Verified"` for the changes you accepted.
+```bash
+sed -i '' 's/origin="Generated by Gradle"/origin="Verified"/g' gradle/verification-metadata.xml
+```
```

Comment:
Let's discuss as part of another PR/ticket if we want to have such a command in the instruction